servers/rendering/renderer_rd/shaders/canvas.glsl

```diff
@@ -26,16 +26,16 @@ layout(location = 3) out vec2 pixel_size_interp;
 
 #endif
 
-#ifdef USE_MATERIAL_UNIFORMS
+#ifdef MATERIAL_UNIFORMS_USED
 layout(set = 1, binding = 0, std140) uniform MaterialUniforms{
-	/* clang-format off */
-MATERIAL_UNIFORMS
+/* clang-format off */
+#MATERIAL_UNIFORMS
 	/* clang-format on */
```

Comment:
Formatting of comment

Reply:
I wonder if these clang-format rules are needed any longer now that I added a #.. @akien-mga?